systemtest/src/test/java/io/strimzi/systemtest/operators/CustomResourceStatusST.java

```diff
@@ -162,72 +164,40 @@ void testKafkaUserStatus(ExtensionContext extensionContext) {
         final TestStorage testStorage = storageMap.get(extensionContext);
         String userName = testStorage.getKafkaUsername();
 
-        resourceManager.createResourceWithWait(extensionContext, KafkaUserTemplates.tlsUser(Constants.TEST_SUITE_NAMESPACE, CUSTOM_RESOURCE_STATUS_CLUSTER_NAME, userName).build());
-
-        LOGGER.info("Checking status of deployed KafkaUser");
-        Condition kafkaCondition = KafkaUserResource.kafkaUserClient().inNamespace(Constants.TEST_SUITE_NAMESPACE).withName(userName).get().getStatus().getConditions().get(0);
-        LOGGER.info("KafkaUser Status: {}", kafkaCondition.getStatus());
-        LOGGER.info("KafkaUser Type: {}", kafkaCondition.getType());
-        assertThat("KafkaUser is in wrong state!", kafkaCondition.getType(), is(Ready.toString()));
-        LOGGER.info("KafkaUser is in desired state: Ready");
-    }
-
-    @ParallelTest
-    void testKafkaUserStatusNotReady(ExtensionContext extensionContext) {
```

Comment:
maybe this PR could be also used to document tests that are modified by these changes.

Reply:
This is very good point, i will do this in separate PR for all the test cases. I think we can finally start to do that in other test suites as well.

Reply:
works for me 👍